### DISCUSSION TITLE
Create an HTML index like on the offical website

### DISCUSSION THREAD

**Discussion by IIPoliII** [squidfunk/mkdocs-material#2769] [Q&A]
I would love to do an awesome index page like this one https://squidfunk.github.io/mkdocs-material/

I tried to look in this repo but it isn't clear on how it's working.
I don't want to manually create my header and footer as well...
I tried to simply write some html/css in my md file but bootstrap is not available and I can't import it because it overrides my theme colours.

So can someone explain to me how this override works?
https://raw.githubusercontent.com/squidfunk/mkdocs-material/1a88966e265abc50b61e17dd9c027434ba08f867/docs/index.md

**Comment by squidfunk** [comment]
Directions were already discussed in: https://github.com/squidfunk/mkdocs-material/issues/1996
